src/test/java/de/unijena/cheminf/fragment/fingerprint/ExampleUsageTest.java

```diff
@@ -0,0 +1,214 @@
+        //Creating a set of names to generate a fingerprint for
+        List<String> tmpMyPartyPeople = List.of("Hugo", "Hannah", "Sam", "Maria");
+        //Generating the bit fingerprint
+        IBitFingerprint tmpMyPartyPeopleFP = tmpFingerprinter.getBitFingerprint(tmpMyPartyPeople);
+        System.out.println(tmpMyPartyPeopleFP.cardinality());
+        /*
+         * Output: 3
+         *
+         * I.e. 3 positive bits in the fingerprint. Maria is ignored since she was not part of the initialisation set.
+         */
+        System.out.println(tmpMyPartyPeopleFP.asBitSet().toString());
+        /*
+         * Output: {0, 1, 3}
+         *
+         * Hannah is represented by position 0, Sam by position 1, and Hugo by position 3 in
+         * the fingerprint and these positions are positive in the bit fingerprint.
+         */
+        //Printing the bit definitions and whether they are positive or not in the "party" set of names
+        for (int i = 0; i < tmpFingerprinter.getSize(); i++) {
+            System.out.println(tmpFingerprinter.getBitDefinition(i) + ": " + tmpMyPartyPeopleFP.get(i));
+        }
+        /*
+         * Output:
+         * Hannah: true
+         * Sam: true
+         * John: false
+         * Hugo: true
+         * Tim: false
+         */
+    }
+    //
+    /**
+     * The intended use case of the fragment fingerprinter functionality is to encode the presence and absence of
+     * substructures in a given molecule that result from a molecular fragmentation study, i.e. the algorithmic
+     * extraction of specific substructures from input molecules. These substructures are automatically extracted and
+     * can be represented by different string-based molecular structure encodings, like SMILES or InChI. Other
+     * key-based substructure fingerprint functionalities require SMARTS strings as inputs and are therefore not
+     * as ubiquitously applicable as the fragment fingerprint for this purpose.
+     *
+     * In the following, a molecular structure data set is imported that contains 100 natural products with a
+     * naphthalene substructure taken from the COCONUT natural products database. These are fragmented using the CDK
+     * ExhaustiveFragmenter functionality that breaks single non-ring bonds in input molecules to generate fragments.
+     * The resulting fragments are collected together with their fraquencies as unique SMILES representations.
+     * Fragments that occur more than two times are then used to initialise the fragment fingerprinter. At the end,
+     * the "naphthalene-derivatives exhaustive fragmenter fingerprint" is generated for 3-hydroxy-2-naphthoic acid.
+     */
+    @Test
+    public void chemicalExampleUsageTest() throws Exception {
+        //Note: make it "path\\to\\coconut_naphthalene_substructure_search_result.sdf" in the example code (I don't want to use the import from test resources here)
```

Comment:
Why not the resources path?

Reply:
My main intention here is to create a kind of basic tutorial for the GitHub wiki. I have put the code into a test class/method "only" to be able to discuss it with you, test it, and have it somewhere in an executable way. In a tutorial, I wouldn't expect anything to be imported from resources. But I guess you're right, I should still do it this way in the test code, will do!